src/player.h

```diff
@@ -166,6 +166,11 @@ class Player
 
 	std::string inventory_formspec;
 
+	bool custom_deathscreen;
+	bool enable_deathscreen;
+	std::string get_deathscreen_formspec() { return deathscreen_formspec; }
```

Comment:
also return a const ref to prevent copying data, this permit to enhance a little bit the performance